packages/@aws-cdk/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -154,6 +161,12 @@ export class Trail extends Resource {
    */
   public readonly trailSnsTopicArn: string;
 
+  /**
+   * The CloudWatch log group to which CloudTrail events are sent.
+   * `undefined` if `sendToCloudWatchLogs` property is false.
+   */
+  public readonly logGroup: logs.ILogGroup | undefined;
```

Comment:
why not just make it optional? instead of the `| undefined`?

Reply:
```suggestion
  public readonly logGroup?: logs.ILogGroup;
```